### ISSUE TITLE
Incorrect fit lines with linetype(qfit) and without reportreg on

### ISSUE THREAD

**Issue by sergiocorreia** [mdroste/stata-binscatter2#10]
Hi Mike,

There is a small bug in the program that triggers with certain options. [Lines 569-574](https://github.com/mdroste/stata-binscatter2/blob/master/binscatter2.ado#L569) of binscatter2 say:

```stata
if "`reg_verbosity'"=="quietly" {
    capture reg `depvar' `x_r2' `x_r' `wt' if `conds', noheader notable
}
else {
    capture noisily reg `depvar' `regressor_list' `wt' if `conds'
}
```

The problem is that `regressor_list` is "x x_squared", but when the regression is run quietly the order of the variables is flipped to "x_squared x", so then when you run predictions (around line 905) the wrong coefficients get used.

Toy example:

```stata
sysuse auto
binscatter2 price weight, line(qfit) // line clearly off and diff than binscatter
binscatter2 price weight, line(qfit) reportreg // line exactly as binscatter
```

Thanks,
Sergio

**Comment by avila** [comment]
I was going to report an issue, but this cover exactly my problem as well... It is open since 2020. Please, let me know if you would like me to attempt a fix and I would gladly write a pull request :)

**Comment by mdroste** [comment]
Pull requests always welcome! Sorry, I haven't had issues turned on in a while. This has been fixed in version 0.23 (and follow-up versions coming soon, to eliminate all easy issues that have built up)

Thanks to both of you!
